test/integration/default/controls/config_spec.rb

```diff
@@ -6,17 +8,29 @@
     it { should be_owned_by 'root' }
     it { should be_grouped_into 'root' }
     its('mode') { should cmp '0644' }
-    its('content') { should include 'This is another example file from SaltStack template-formula.' }
-    its('content') { should include '"added_in_pillar": "pillar_value"' }
-    its('content') { should include '"added_in_defaults": "defaults_value"' }
-    its('content') { should include '"added_in_lookup": "lookup_value"' }
-    its('content') { should include '"config": "/etc/template-formula.conf"' }
+    its('content') do
+      should include(
+        'This is another example file from SaltStack template-formula.'
+      )
+    end
+    its('content') { should include('"added_in_pillar": "pillar_value"') }
```

Comment:
Can I point out that the [Inspec style guide](https://www.inspec.io/docs/reference/style/) recommends avoiding unnecessary parentheses in matchers.
> Adding additional parentheses is not required and provides more readability if it is not used
Personally I would agree with that.

Reply:
@dafyddj Point understood.  My only reason for doing it was for consistency with the lines that had to be split over multiple lines.  Since there's going to be a lot of this throughout the formulas, what method would you suggest?  It's a shame the guide doesn't give guidelines for splitting long lines.

@n-rodriguez Any thoughts, from a general Ruby perspective?

Reply:
I'm not sure - I'm no Ruby expert. I would need to look into the options.

Reply:
@dafyddj Since this isn't a critical part of this PR, I'm happy to adjust to the style guide.  I've already made the changes locally and I'll push them here in due course.  Thanks for pointing that out in the first place.

Reply:
Done.